erpnext/hr/doctype/job_applicant/job_applicant.py

```diff
@@ -26,20 +26,11 @@ def autoname(self):
 		self.name = " - ".join(keys)
 
 	def validate(self):
-		self.applicant_name = self.first_name + " " + self.last_name
-		self.check_email_id_is_unique()
```

Comment:
why are we removing this check?

Reply:
There is a property in DocField to set the field unique. as this is an outdated code so, I removed that and checked the property to is unique. Please refer to the screenshot I had made it unique from here.
![Screenshot from 2021-09-22 12-08-02](https://user-images.githubusercontent.com/32095923/134294777-ccd5c406-f7de-4010-8674-3e04b6c610be.png)
.